src/renderers/index.ts

```diff
@@ -31,7 +32,8 @@ export
 class HTMLWidget extends Widget {
   constructor(html: string) {
     super();
-    this.node.innerHTML = html;
+    let range = document.createRange();
+    this.node.appendChild(range.createContextualFragment(html));
```

Comment:
We noticed that Safari, at least, doesn't run `<script>` elements, similar to when using `innerHTML`. It's also listed as "experimental" on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/range/createContextualFragment). I think a better solution would be to:

1. Create a document fragment from the string, setting content using `innerHTML`.
2. Pull out all `<script>` elements.
3. When widget is attached to DOM, run the saved `<script>`s.

I believe jQuery does something along these lines.